native/src/main/java/io/ballerina/stdlib/crypto/nativeimpl/Decode.java

```diff
@@ -253,4 +320,8 @@ public static Object buildRsaPublicKey(BString modulus, BString exponent) {
             return CryptoUtils.createError("Algorithm of the key factory is not found: " + e.getMessage());
         }
     }
+
+    public static Object generateBCryptHash(BArray input, int cost) {
```

Comment:
Are we using this method anywhere? Are we not using the input arguments in method body?

Reply:
No, it is not being used. It was a mistake adding it to this PR. :) Removed in [6f1b573](https://github.com/ballerina-platform/module-ballerina-crypto/pull/533/commits/6f1b5734ec97d4e3a5a4e254f44fe11bf72a1202)